contracts/reviewer.sol

```diff
@@ -29,22 +52,60 @@ contract ReviewerTransactionGuard is BaseGuard {
         _;
     }
 
-    address public ApplicationReviewRegistryAddress = address(); // Enter Registry Address according to the chain this contract is deployed on
-    address public ApplicationRegistryAddress = address(); // Enter Registry Address according to the chain this contract is deployed on
-    address public WorkspaceRegistryAddress = address(); // Enter Registry Address according to the chain this contract is deployed on
+    modifier onlyOwner() {
```

Comment:
I don't think we need an onlyOwner modifier. The address changes for interfaces should be approved by all Safe owners and hence should have an onlySafe modifier.